polygeist/include/mlir/Dialect/Polygeist/Analysis/ReachingDefinitionAnalysis.h

```diff
@@ -152,7 +162,78 @@ class ReachingDefinitionAnalysis
 
 private:
   DenseMap<FunctionOpInterface, std::unique_ptr<AliasOracle>> aliasOracles;
-  mlir::AliasAnalysis &aliasAnalysis;
+  AliasAnalysis &aliasAnalysis;
+};
+
+/// A class representing the value a formal argument of a function has the
+/// caller of the function.
```

Comment:
```suggestion
/// A class representing the value a formal argument of a function has in the
/// caller of the function.
```
? not sure I understand what this comment means.

Reply:
done.